spec/stack_spec.rb

```diff
@@ -45,6 +45,29 @@
     }.to raise_error(Aws::CloudFormation::Errors::ValidationError, /does not exist/)
   end
 
+  it 'creates a stack with tags' do
+    name = "spec-aws-ruby-stack-create-tags"
+    stack = new_stack(name: name, filename: "templates/simple.yml", overrides: {tags: {foo: "bar"}})
+    stack.create(params: {}, wait: true)
+
+    tags = ::Aws::CloudFormation::Stack.new(name: name, client: cfn_client).tags
+    expect(tags[0].key).to eq "foo"
+    expect(tags[0].value).to eq "bar"
+  end
+
+  it 'updates a stack with new tag values' do
+    name = "spec-aws-ruby-stack-update-tags"
+    stack = new_stack(name: name, filename: "templates/simple.yml", overrides: {tags: {foo: "bar"}})
+    stack.create(params: {}, wait: true)
+
+    stack = new_stack(name: name, filename: "templates/simple.yml", overrides: {tags: {foo: "howdy"}})
```

Comment:
stack should also be a let

Reply:
stack is set 3 times in this example to 3 different values, can't be a `let`